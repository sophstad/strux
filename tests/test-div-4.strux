int main() {
	num three = 3.0;
	num ten = 10.0;
	print(three / ten);
	print(ten / three);
	print(three + ten);
	print(ten + three);
	print(ten - three);
	print(ten % three);
	print(three % ten);
	return 0;
}