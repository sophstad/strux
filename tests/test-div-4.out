0.300000
3.333333
13.000000
13.000000
7.000000
1.000000
3.000000
